impl/client/src/main/javascript/web/dojo/pentaho/common/FieldList.js

```diff
@@ -311,6 +312,7 @@ define(["dojo/_base/declare", "dijit/_WidgetBase", "dijit/_Templated", "dojo/on"
     calc.elementType = pentaho.pda.Column.ELEMENT_TYPES.CATEGORY;
     calc.dataType = pentaho.pda.Column.DATA_TYPES.NONE;
     calc.name = "Calculated Fields";
+    calc.class="CAT_CALC_FIELD";
```

Comment:
we can use the constant created in controller.js
`pentaho.pda.Column.ELEMENT_TYPES.CAT_CALC_FIELD = "CAT_CALC_FIELD";`